backend/src/main/java/com/bakdata/conquery/sql/conversion/dialect/SqlFunctionProvider.java

```diff
@@ -65,6 +65,11 @@ public interface SqlFunctionProvider {
 
 	Condition likeRegex(Field<String> field, String pattern);
 
+	/**
+	 * @return The numerical year and quarter of the given date column as "yyyy-Qx" string expression with x being the quarter.
+	 */
+	Field<String> quartersPerYear(Field<Date> dateField);
```

Comment:
der name ist recht ungeschickt gewählt, ich hab das einfach YearQuarter genannt. quartersInYear ist ein anderer Select, der die Quartale in einem Jahr zählt.

Reply:
@jnsrnhld der PR ist noch offen, kannst du das bitte fixen, dann kann auch gemerged werden.

Reply:
Sorry, ich dachte ich hätte das längst gemacht - hab es jetzt nachgeholt!